docs/UserGuide.md

```diff
@@ -408,46 +409,86 @@ DELIVERY_DATE)!_
 
-**Example:** `delivery list --status created --customer 1 --date 2023-12-12 --sort desc`
+**Format:**
+`delivery list [--status STATUS] [--customer CUSTOMER_ID] [--date EXPECTED_DELIVERY_DATE]  [--sort SORT]`{.swift}
 
-**Accepted Values:**
+<box type="note" background-color="#dff0d8" border-color="#d6e9c6" icon=":information_source:">
 
-_STATUS_: CREATED/SHIPPED/COMPLETED/CANCELLED. If unspecified, defaults to show all deliveries.
+**Note:**
 
-_CUSTOMER_ID_: Integer
+* All fields are optional.
+* The delivery `STATUS`{.swift} accepts the following values: `CREATED`{.swift}/`SHIPPED`{.swift}/`COMPLETED`{.swift}/
+  `CANCELLED`{.swift}.
+* The `CUSTOMER_ID`{.swift} must be an integer.
+* The `EXPECTED_DELIVERY_DATE`{.swift} must be in yyyy-MM-dd format or `TODAY`{.swift} for today’s date.
+* The `SORT`{.swift} accepts the following values: `ASC`{.swift}/`DESC`{.swift}.
+  </box>
 
-_EXPECTED_DELIVERY_DATE_: Expected Delivery Date String in YYYY-MM-DD format or `today` for today’s date
+**Examples:**
 
-_SORT_: String of either `asc` for ascending or `desc` for descending or defaults to sort by expected delivery date.
+* `delivery list --status CREATED --customer 1 --date 2023-12-12 --sort DESC`{.swift} </br>
+  Lists all deliveries with status `CREATED`{.swift} for customer with ID `1`{.swift} and expected delivery date of 12
+  Dec 2023 in descending order.
 
-**Command succeeds (>0 deliveries):**
-![](images/delivery/delivery_list.png)
+* `delivery list --status SHIPPED --date TODAY`{.swift} </br>
+  Lists all deliveries with status `SHIPPED`{.swift} for all customers and expected delivery date of today in ascending
+  order.
 
-**Command failed (0 deliveries):** _There are currently no deliveries!_
+<box type="tip" background-color="#d9edf7" border-color="#bce8f1" icon=":bulb:">
 
-### View details of deliveries
+**Tip:**
+You combine any of the filters and sort options to get the list of deliveries that you want.
+</box>
 
-Shows the details of the specified delivery.
+**Before:**
 
-**Format:** `delivery view DELIVERY_ID`
+![](images/customer/customerList.png)
 
-**Example:** `delivery view 1001`
+**After:**
+![](images/delivery/DeliveryListSuccess.png)
 
-**Accepted Values:**
+[&uarr; Back to Table of Contents](#top)
 
-_DELIVERY_ID_: Integer
+### View details of a delivery
 
-**Command succeeds:**
+> Shows the details of the specified delivery.
+
+If you would like to get the details of a specific delivery, you can use `delivery view`{.swift} to view the details of
+the delivery. You can see the delivery id, delivery name, delivery status, address, customer name customer ID, ordered
+date, expected delivery date, and notes (if any).
+
+**Format:** `delivery view DELIVERY_ID`{.swift}
+
+<box type="note" background-color="#dff0d8" border-color="#d6e9c6" icon=":information_source:">
+
+**Note:**
+
+* All fields are required.
+* The `DELIVERY_ID`{.swift} must be an integer.
+  </box>
+
+**Example:**
+
+* `delivery view 1001`{.swift} </br>
```

Comment:
Would it be more apt to show delivery view 1 instead inline with the success image